### DISCUSSION TITLE
"Select more Photos" iOS 14

### DISCUSSION THREAD

**Discussion by b3nni97** [fluttercandies/flutter_wechat_assets_picker#92] [Q&A] [answered]
Hi guys, I have a problem with selecting pictures and videos on IOS. The first time when I want to select pictures comes the choice if I want to keep the current selection or select more photos. 
Unfortunately this selection comes only 1 time and the 2nd time I select I can not select more photos from my gallery.
How can I trigger the "Select more photos" manually?

**Comment by AlexV525** [comment, marked as answer]
With the new 6.0 version release, you can either toggle `presentLimited` if your permission is limited currently, by tapping the button in the path entities list, or change granted permission by tapping the tip below the picker. See the example for more info.
